frontend/src/components/Action/QuestionActionsList.tsx

```diff
@@ -18,58 +25,67 @@ interface Props {
 }
 
 const QuestionActionsList = ({ question, participants, onActionCreate }: Props) => {
-    const [isActionCreateSidebarOpen, setIsActionCreateSidebarOpen] = useState<boolean>(false)
+    const [isSidebarOpen, setIsSidebarOpen] = useState<boolean>(false)
+    const [actionToEdit, setActionToEdit] = useState<Action>()
     const actions = question.actions
 
-    return <>
-        <Container>
-            <Box pl={15}>
-                <Box display="flex" alignItems="center">
-                    <Box flexGrow={1} display="flex" flexDirection="column-reverse">
-                        <Typography variant="body_short" bold>Actions</Typography>
-                    </Box>
-                    <Box >
-                        <Button
-                            variant="ghost"
-                            onClick={() => setIsActionCreateSidebarOpen(true)}
-                        >
-                            <Icon data={add}></Icon>
-                            Add action
-                        </Button>
+    const editAction = (action: Action) => {
+        setIsSidebarOpen(true)
+        setActionToEdit(action)
+    }
+
+    const onClose = () => {
+        setIsSidebarOpen(false)
+        actionToEdit && setActionToEdit(undefined)
```

Comment:
I cannot remember having seen this used like this before, cool! But I find it more readable and explicit like this: 
```suggestion
        if (actionToEdit !== undefined) {
            setActionToEdit(undefined)
        }
```

I wanted to say that, but if you like that better I might develop a more pronounced preference later.

Reply:
Thanks, I agree! I like to use this notation when I check for presence before using an attribute on an object, but in this case, since it's actually checking if it should perform an action, it's better to use the explicit way.

Also, I just found that the way I normally use it (`user && user.name` for instance), can be better written like user?.name, which is pretty cool :) (more info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining))